### PR TITLE
Create a 'benchmarks' extra, separate from 'dev'

### PR DESCRIPTION
**Pull request by ryanjulian**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -75,13 +70,23 @@
...
     'recommonmark',
-    'rlkit @ https://{}@api.github.com/repos/vitchyr/rlkit/tarball/1d469a509b797ca04a39b8734c1816ca7d108fc8'.format(GARAGE_GH_TOKEN),  # noqa: E501
-    'seaborn',
     'sphinx==2.4.4',
     'sphinx_rtd_theme',
     'yapf==0.28.0',
 ]  # yapf: disable
 
+# Benchmark dependencies (*not* included in 'all')
+EXTRAS['dev'] = [
```
Comment (gitanshu) [inline setup.py:79]:
```suggestion
EXTRAS['benchmarks'] = [
```

**Comment by ryanjulian** [comment]
@zequnyu can you link to the code you're talking about?

**Comment by zequnyu** [comment]
https://github.com/rlworkgroup/garage/blob/1016d58d7bfe3b7ec90e7a05e40839699653219d/tests/benchmark_helper.py#L93

to

https://github.com/rlworkgroup/garage/blob/master/src/garage/_functions.py

**Comment by krzentner** [comment]
> Do you think `garage/_functions.py` is a good place?

I don't really think that's a good place for this function. Preferably every function directly under `garage` should have an obvious purpose for most users, but this function is really only relevant for how we manipulate experiment results.

**Comment by zequnyu** [comment]
@Mergifyio rebase

**Comment by ryanjulian** [comment]
@zequnyu can you finish this PR? Do you need any extra information?

**Comment by zequnyu** [comment]
@Mergifyio rebase

**Comment by ryanjulian** [comment]
@zequnyu do you know how to finish this PR?

**Comment by zequnyu** [comment]
@ryanjulian Can you force push this? I deleted some functions in `tests.helper` but not familiar with the rest. It's just the pre-commit doc check is blocking me.

File: benchmarks/setup.py [hunk 2]
```diff
@@ -2,9 +2,19 @@
...
+import os
 
 from setuptools import find_packages, setup
 
+GARAGE_GH_TOKEN = os.environ.get('GARAGE_GH_TOKEN') or 'git'
+
+REQUIRED = [
+    # Please keep alphabetized
+    'baselines @ https://{}@api.github.com/repos/openai/baselines/tarball/ea25b9e8b234e6ee1bca43083f8f3cf974143998'.format(GARAGE_GH_TOKEN),  # noqa: E501
+    'matplotlib',
```
Comment (ryanjulian) [inline benchmarks/setup.py:14]: i notice you removed several deps -- are these no longer needed in the benchmark scripts?

Comment (zequnyu) [reply benchmarks/setup.py:14]: I checked the codebase and they are not used anywhere.

**Pull request merged**